CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 Records breaking changes from major version bumps
 
+## 10.4.0
```

Comment:
No changelog record needing for a non breaking change. Your commit message should be the appropriate place for context I think.